doc/whatsnew/2/2.14/summary.rst

```diff
@@ -256,6 +256,11 @@ Other Changes
 
   Relates to #6612
 
+* Don't report ``unsupported-binary-operation`` on Python <= 3.9 when using the ``|`` operator
+  with types, if one has a metaclass that overloads ``__or__`` or ``__ror__`` as appropriate.
+
+  Closes #4951
+
```

Comment:
Unfortunately we need to still add the changelog in full.rst too until we release 2.14.0.

Reply:
Now I've missed the 2.14 cut-off I guess I have to move this. I've put the note in `index.rst` in the 2.15 section because that's where all the section headings are, but I don't know if this is going to get moved into separate files later?